bot/connector-whatsapp-cloud/src/main/kotlin/model/send/message/WhatsAppCloudSendBotImageMessage.kt

```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
  */
 
-package ai.tock.bot.connector.whatsapp.cloud.model.send.media
+package ai.tock.bot.connector.whatsapp.cloud.model.send.message
 
-import ai.tock.bot.connector.whatsapp.cloud.model.send.message.WhatsAppCloudBotRecipientType
-import ai.tock.bot.connector.whatsapp.cloud.model.send.message.WhatsAppCloudSendBotMessage
-import ai.tock.bot.connector.whatsapp.cloud.model.send.message.WhatsAppCoudBotMessageType
+import ai.tock.bot.connector.whatsapp.cloud.model.common.TextContent
+import ai.tock.bot.connector.whatsapp.cloud.model.send.message.content.WhatsAppCloudBotImage
+import com.fasterxml.jackson.annotation.JsonProperty
 
-data class WhatsAppCloudSendBotImageMessage(
+data class WhatsAppCloudSendBotImageMessage (
         override val messagingProduct: String,
-        override val to: String?,
-        override val recipientType: WhatsAppCloudBotRecipientType?
-) : WhatsAppCloudSendBotMessage(WhatsAppCoudBotMessageType.image) {
-
-}
+        val image: WhatsAppCloudBotImage,
+        override val recipientType: WhatsAppCloudBotRecipientType,
+        override val to: String,
+) : WhatsAppCloudSendBotMessage(WhatsAppCoudBotMessageType.image)
```

Comment:
Semi-related to this PR: this class name has a typo

Reply:
je ne pense pas

Reply:
`WhatsAppCoudBotMessageType` instead of `WhatsAppCloudBotMessageType`?